database/tables/pl_sessions.pg

```diff
@@ -0,0 +1,7 @@
+columns
+    session: jsonb
+    sid: text not null
+    updated_at: timestamp with time zone not null default CURRENT_TIMESTAMP
```

Comment:
We should index this column to minimize the likelihood of doing a table scan for operations like `all`, `length`, or the expiration cronjob.

Reply:
Sure. I didn't realize we had to index `tstz` tables; I thought postgres did that for us?

Reply:
Nope; indexes have to be explicitly created if they're desired.